packages/core/src/common/keybinding.spec.ts

```diff
@@ -211,9 +247,9 @@ export class TestContribution implements CommandContribution, KeybindingContribu
     }
 
     registerKeybindings(keybindings: KeybindingRegistry): void {
-        const keycode1 = KeyCode.parse('ctrl+a');
-        const keycode2 = KeyCode.parse('ctrl+f1');
-        const keycode3 = KeyCode.parse('ctrl+f2');
+        const keycode1 = KeyCode.parse('shift+a');
```

Comment:
Is it really valid to have a keybinding with shift+a? I mean, this would produce normally uppercase 'A'...

Reply:
It's technically not valid as in a usability sense indeed, but the way keybindings currently work is that you could technically rebind the normal shift-a behavior to whatever you'd like, so the tests would work in that sense. I can probably change the keybindings to something that makes more sense though.

Reply:
Bah, I think it's fine.